sample_apps/run_example.sh

```diff
@@ -56,6 +74,12 @@ Jq=jq    # Needed for OE app; will be established later on.
 MbedTLS_dir="mbedtls"
 Simple_App_under_gramine_MbedTLS_dir="sample_apps/simple_app_under_gramine/${MbedTLS_dir}"
 
+# --------------------------------------------------------------------------------
+# sample_app_under_cca uses ISLET SDK, which needs to be downloaded. To avoid
+# downloading repos frequently, provide this global variable to see if it has
+# been downloaded before.
+ISLET_ROOT="${CERT_PROTO}/third_party/islet"
```

Comment:
Adding external dependency.

Nothing major has changed in this script. I have annotated places which merit a closer look during review.